### PR TITLE
clean: remove 3rd party links

### PR DESCRIPTION
**Pull request by raelga**
- Removes 3rd party links from the Spanish Daemonset documentation.
- Minor format changes

Related to https://github.com/kubernetes/website/pull/21531 and https://github.com/kubernetes/website/issues/20232

/kind cleanup
/assign

### CODE REVIEW

File: content/es/docs/concepts/workloads/controllers/daemonset.md [hunk 1]
```diff
@@ -86,15 +77,15 @@ Cuando se configura ambos campos, el resultado es conjuntivo (AND).
...
 Si se especifica el campo `.spec.selector`, entonces debe coincidir con el campo `.spec.template.metadata.labels`. Aquellas configuraciones que no coinciden, son rechazadas por la API.
 
-Además, normalmente no se debería crear ningún Pod con etiquetas que coincidan con el selector, bien sea de forma directa, via otro 
+Además, normalmente no se debería crear ningún Pod con etiquetas que coincidan con el selector, bien sea de forma directa, via otro
 DaemonSet, o via otro controlador como un ReplicaSet.  De ser así, el controlador del DaemonSet
-pensará que dichos Pods fueron en realidad creados por él mismo. Kubernetes, en cualquier caso, no te impide realizar esta 
+pensará que dichos Pods fueron en realidad creados por él mismo. Kubernetes, en cualquier caso, no te impide realizar esta
 operación. Un caso donde puede que necesites hacer esto es cuando quieres crear manualmente un Pod con un valor diferente en un nodo para pruebas.
 
 ### Ejecutar Pods sólo en algunos Nodos
```
Comment (electrocucaracha) [inline content/es/docs/concepts/workloads/controllers/daemonset.md:85]: Parece ser que [este](https://github.com/kubernetes/website/pull/21531/files) cambio tambien se realizo
```suggestion
### Ejecutar Pods sólo en Nodos seleccionados
```

**Review by electrocucaracha — CHANGES_REQUESTED** [review]

File: content/es/docs/concepts/workloads/controllers/daemonset.md (hunk 1 above)
Comment (raelga) [reply content/es/docs/concepts/workloads/controllers/daemonset.md]: Añadido @electrocucaracha

**Pull request merged**